packages/rrweb/src/record/index.ts

```diff
@@ -49,11 +49,11 @@ function wrapEvent(e: event): eventWithTime {
 
 let wrappedEmit!: (e: eventWithTime, isCheckout?: boolean) => void;
 
-let takeFullSnapshot!: (isCheckout?: boolean) => void;
+let _takeFullSnapshot!: (isCheckout?: boolean) => void;
 let canvasManager!: CanvasManager;
 let recording = false;
 
-const mirror = createMirror();
+export const mirror = createMirror();
```

Comment:
🤔 

so rrweb1 did have `mirror` exported directly, but they deprecated that -- I'm not sure the reasoning (it's possible they wanted to keep compatibility and decided to have a different interface to `mirror`, but maybe worthwhile to dig in and see why they changed public interface to mirror.

Reply:
yeah so that mirror was different - it was a shared mirror for recording/replaying, I believe (also that mirror was a noop by now).

This is different in that we just always export the recording mirror from `@sentry/rrweb`.